science/gmtsar-data/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gmtsar-data
+version             1.0
```

Comment:
Is the data actually tagged with this `1.0` version number, or did you just make it up? If you made it up, then I suggest changing it to a *YYYYMMDD* value, such as the file's modification date, 20150202.